### DISCUSSION TITLE
Including Vizzu for animate visualizations

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [quarto-dev/quarto-cli#620] [Q&A] [answered]
Hi all,

I just came across [Vizzu](https://vizzuhq.com) and thought that integrating it with Quarto would be awesome.

Cheers.

**Comment by jjallaire** [comment, marked as answer]
Thanks! The way this would currently work is that someone creates a Jupyter Widget (https://jupyter.org/widgets) or R htmlwidget (https://htmlwidgets.org) to wrap Vizzu (both of these would then work in Quarto).

There is also the possibility of using Vizzu directly w/ Observable (https://quarto.org/docs/interactive/ojs/)

**Reply by camilogarciabotero** [reply]
I'll definitely take a look! Thanks!
